pkg/portal/graphql/app.go

```diff
@@ -26,13 +28,37 @@ var nodeApp = node(
 			"appConfig": &graphql.Field{
```

Comment:
I suggest we rename the fields from `appConfig`, `appConfigEffective` to `rawAppConfig` and `effectiveAppConfig`